external-import/mandiant/.gitignore

```diff
@@ -0,0 +1 @@
+src/mandiant/data
```

Comment:
This could be in a `.dockerignore` file as well to avoid copying data into the image by mistake.

Reply:
Actually, that .gitignore was just for me on my dev env, it should't been commited.